src/main/java/Main/Main.java

```diff
@@ -0,0 +1,39 @@
+package Main;
+
+import Controller.Controller;
+import Model.Board;
+import Model.FEN;
+import Observer.Observer;
+import View.MainView;
+import javafx.application.Application;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import static java.lang.Thread.sleep;
+
+public class Main extends Application {
+
+    @Override
+    public void start(Stage stage) throws IOException, InterruptedException {
+
+
+
+        Observer ObserverMainView = new MainView(stage); // Start up View
+
+
+        Board board = new Board(ObserverMainView); // Start up Model
+
+        MainView mainView = (MainView) ObserverMainView;
+
+        Controller controller = new Controller(board, mainView); // Start up Controller
```

Comment:
Nowhere here do you actually 'start' the application. it starts by instantiating an instance of `MainView` which in turn calls `stage.show()`. In a way, the game only really works by 'luck' because by the time you click on the first piece, it has luckily already finished initialising the controller. 

For example, what if your on a slow machine and the controller is still running the `addOnClick()` method, and the player clicks a square that hasn't yet been reached.. it would not do anything. (i mean its so fast that its basically impossible to replicate that scenario.. but you get the idea).

More clean here would be to remove these two lines from the constructor of `MainView`
```java
stage.setScene(scene);
stage.show();
```
and instead place them in a method called by the controller, and add something like this as the final line in this `start()` method:
```java
controller.showBoard()
```

Reply:
Implemented. Is it worth making Controller the main class or is that not how to do it?

Reply:
Kept the Main Class and tried to keep it as simple as possible